dockerspawner/dockerspawner.py

```diff
@@ -69,6 +80,22 @@ def client(self):
         )
     )
 
+    tls = Bool(False, config=True, help="If True, connect to docker with --tls")
+    tls_verify = Bool(False, config=True, help="If True, connect to docker with --tlsverify")
+    tls_ca = Unicode("", config=True, help="Path to CA certificate for docker TLS")
+    tls_cert = Unicode("", config=True, help="Path to client certificate for docker TLS")
+    tls_key = Unicode("", config=True, help="Path to client key for docker TLS")
```

Comment:
Yeah, it probably would be good to document that. It corresponds essentially to this: https://docs.docker.com/articles/https/#client-modes , so the valid modes are:

* `tls=True`
* `tls_verify=True` and `tls_ca` is defined
* `tls_verify=True` and `tls_ca`, `tls_cert`, and `tls_key` are defined
* `tls_cert` and `tls_key` are defined

I think any other combination will cause docker-py to complain.